bats/os_helper.bash

```diff
@@ -57,6 +57,10 @@ tIsRHEL() {
   fi
 }
 
+tIsRHEL() {
+  tIsEL "$1" && grep -q 'Red Hat Enterprise Linux' /etc/redhat-release
```

Comment:
`ID` in `/etc/os-release` is more reliable because of false positives on Oracle Linux. On an Oracle Linux 8 container:
```console
# cat /etc/redhat-release 
Red Hat Enterprise Linux release 8.7 (Ootpa) 
```